appinventor/blocklyeditor/src/blocks/lists.js

```diff
@@ -790,6 +790,61 @@ Blockly.Blocks['lists_minimum_number'] = {
   category : 'Lists',
   helpUrl : Blockly.Msg.LANG_LISTS_MIN_NUMBER_HELPURL,
   init : function() {
+      this.setColour(Blockly.LIST_CATEGORY_HUE);
```

Comment:
There's too much indentation here. This line and the ones following should only be +2.